browser-bunyan/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```

Comment:
I could not get your export suggestion to work with this DefinitelyTyped Repo.

Below works in our local project. It nor your suggestion (from "bunyan") would not pass Travis CI.

```
  export * from "@types/bunyan
```